core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -163,8 +163,11 @@ protected[controller] class RestAPIVersion_v1(
 
     // initialize backend services
     protected implicit val consulServer = WhiskServices.consulServer(config)
-    protected implicit val entitlementService = WhiskServices.entitlementService(config)
-    protected implicit val (performLoadBalancerRequest, getInvokerHealth, queryActivationResponse) = WhiskServices.makeLoadBalancerComponent(config)
+    protected implicit val loadBalancer = WhiskServices.makeLoadBalancerComponent(config)
+    protected implicit val performLoadBalancerRequest = loadBalancer.acceptRequest _
+    protected implicit val getInvokerHealth = loadBalancer.getInvokerHealth _
```

Comment:
Do we need this? I think this can be inlined as `loadBalancer.getInvokerHealth()` in `internalInvokerHealth` at the bottom of this file.